content/en/user-guide/lambda-tools/hot-reloading/index.md

```diff
@@ -40,6 +40,17 @@ Therefore, filesystem changes persist between code changes for invocations dispa
 If using Docker Desktop on macOS, you might need to allow [file sharing](https://docs.docker.com/desktop/settings/mac/#file-sharing) for your target folders.
 MacOS may prompt you to grant Docker access to your target folders.
 
+**WSL2-compatible paths required with Rancher Desktop on Windows:**
+Make sure your Lambda handler paths are specified using WSL2-compatible paths. For example, instead of using a Windows-style path like:
```

Comment:
```suggestion
Make sure your Lambda handler paths are specified using WSL2-compatible paths. For example, instead of using a Windows-style path such as:
```